### PR TITLE
docs(ideas): add picture finder

### PR DESCRIPTION
**Pull request by MahendraBishnoi29**

### CODE REVIEW

**Comment by MahendraBishnoi29** [comment]
@ming-tsai review my PR please.

File: data/ideas.json [hunk 1]
```diff
@@ -185,6 +185,12 @@
         "level": "advanced",
         "tags": "roadmap, success, process"
     },
+    {
+        "title": "Picture Finder",
+        "description": "A website where we search for any picture that is available on unsplash. we can use unsplash API for this, also we can use axios for fetching the data and a UI library for styling.",
+        "level": "beginner",
+        "tags": "api, react"
```
Comment (ming-tsai) [inline data/ideas.json:192]: - The first tag is used to match projects with an idea, so please add the book-app tag
- Remove the programming tag, due to is only an idea so the everyone could make on whatever programming language
```suggestion
        "tags": "picture-finder, api"
```

**Review by ming-tsai — CHANGES_REQUESTED** [review]
Hi @MahendraBishnoi29,
Some suggestions for the pull request.

File: data/ideas.json [hunk 2]
```diff
@@ -185,6 +185,12 @@
         "level": "advanced",
         "tags": "roadmap, success, process"
     },
+    {
+        "title": "Picture Finder",
+        "description": "A website where we search for any picture that is available on unsplash. we can use unsplash API for this, also we can use axios for fetching the data and a UI library for styling.",
+        "level": "beginner",
```
Comment (ming-tsai) [inline data/ideas.json:191]: The level should be one of this list  `basic, intermediate, advanced`
```suggestion
        "level": "basic",
```

**Review by ming-tsai — CHANGES_REQUESTED** [review]

**Review by ming-tsai — APPROVED** [review]

**Pull request merged**